lib/decorators-serialization.js

```diff
@@ -1,9 +1,51 @@
 'use strict';
 
-module.exports = {
+const mdsf = require('mdsf');
+
+const objFilter = (obj, blockedKeys) => {
+  if (!blockedKeys) return obj;
+  return Object.keys(obj).reduce((newObj, key) => {
+    if (!blockedKeys.includes(key)) newObj[key] = obj[key];
```

Comment:
I'd prefer to convert `blockedKeys` to `Set` and use `has` instead of `includes`.